### DISCUSSION TITLE
When will we get the first community release?

### DISCUSSION THREAD

**Discussion by weavermedia** [SynthstromAudible/DelugeFirmware#83] [Q&A] [answered]
I'm looking forward to trying out the new LFO and sync features! 😀

**Comment by litui** [comment, marked as answer]
"Releases" are not a decided matter yet. Community is an ongoing moving target as folks work to fix and tidy things, establish how governance and contribution will work, etc. In the meantime, if you're impatient, you can always pester someone working on the project for a developer release build.

**Reply by weavermedia** [reply]
Gotcha. I was surprised to see actual features added this quickly to the community code. I expected bug fixes, restructuring and general tidying up but not new LFO and sync options! 😀

**Reply by weavermedia** [reply]
I didn't realize DBT was up and running for the Mac. I can build my own releases now. Thanks for all the hard work 🙌

**Reply by jamiefaye** [reply]
I would propose a regular release schedule, like every 3 months with a beta period and so on. We would also have "nightly builds" for the adventurous.

**Reply by jamiefaye** [reply]
As for the first one, I would like to see us reorganize the code into directories and eliminate all the compile warnings.

**Reply by litui** [reply]
I like the nightly builds idea. Per-merge has been suggested in the community/governance PR but I personally think that's too frequent (especially when not all merges will even contain core code changes).

**Reply by weavermedia** [reply]
Agreed on not doing per-merge. Not only for those reason but also nightly gives any buggy merges a chance to get fixed later in the day before the nightly.

**Reply by alex-r-t** [reply]
@jamiefaye 

> As for the first one, I would like to see us reorganize the code into directories and eliminate all the compile warnings.

Is there any concern from Rohan about engaging in that level of refactoring (directory structure, not the warnings)? Ideally, I'd think we'd want to have the code we write at least somewhat portable to the official Deluge codebase, but ultimately that's up to folks more on the Synthstrom-side of things. I'm not sure how tightly the "Official" and "Community" repos should be.
